packages/edit-site/src/components/layout/index.js

```diff
@@ -226,13 +225,6 @@ export default function Layout() {
 					animate={ headerAnimationState }
 				>
 					<SiteHub
-						variants={ {
```

Comment:
I don't understand why this variants is a prop of SiteHub and not applied directly within the component.